docs/platform/concepts/permissions.md

```diff
@@ -40,6 +40,8 @@ and services within it.
 | Manage domains              | `organization:domains:write`   | <ul> <li> Add, edit, and remove domains. </li> <li> View all organization domains. </li> </ul>                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | Manage groups               | `organization:groups:write`    | <ul> <li> Create, edit, and delete groups. </li> <li> Add organization and application users to groups. </li> <li> Remove organization and application users from groups. </li> </ul>                                                                                                                                                                                                                                                                                                                          |
 | Manage projects             | `organization:projects:write`  | <ul> <li> Create and delete projects. </li> <li> Assign projects to billing groups. </li> <li> Add and remove project tags. </li> </ul> **Cannot otherwise access or move the project or its services.**                                                                                                                                                                                                                                                                                           |
+| View organization networking    | `organization:networking:read`   | <ul> <li> Read all organization VPCs. </li> </ul>                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I forgot to mention when we talked about these permissions that they've been listed in alphabetical order by API name. There's a comment in this doc now, but the networking permissions should go after projects. :)